src/include/daos_srv/control.h

```diff
@@ -14,21 +14,80 @@
 #include <stdint.h>
 #include <stdbool.h>
 #include <assert.h>
+#include <string.h>
 
-/*
+/**
  * Space separated string of CLI options to pass to DPDK when started during
  * spdk_env_init(). These options will override the DPDK defaults.
  */
 extern const char *
 dpdk_cli_override_opts;
 
-/* Device state flags */
+/** Device state flags */
 #define NVME_DEV_FL_PLUGGED	(1 << 0)
 #define NVME_DEV_FL_INUSE	(1 << 1) /* Used by DAOS (present in SMD) */
 #define NVME_DEV_FL_FAULTY	(1 << 2)
 #define NVME_DEV_FL_IDENTIFY	(1 << 3) /* SSD being identified by LED activity */
 
-/*
+/** Device state combinations */
+#define NVME_DEV_STATE_NORMAL (NVME_DEV_FL_PLUGGED | NVME_DEV_FL_INUSE)
+#define NVME_DEV_STATE_FAULTY (NVME_DEV_STATE_NORMAL | NVME_DEV_FL_FAULTY)
```

Comment:
As requested by @sydneyvanda ,  I've reduced the combinations to a single string and so when we unpack to a state from a string we lose some information (as discussed) so the state combination that will result from "EVICTED" status string is PLUGGED|INUSE|FAULTY because we lost the INUSE flag when squashing to a single string